sdcm/nemesis.py

```diff
@@ -1074,6 +1074,35 @@ def disrupt_modify_table(self):
         disrupt_func = getattr(self, disrupt_func_name)
         disrupt_func()
 
+    def disrupt_mgmt_backup(self):
+        self._set_current_disruption('ManagementBackup')
+        if not self.cluster.params.get('use_mgmt', default=None):
+            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+
+        manager_node = self.monitoring_set.nodes[0]
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=manager_node)
+
+        cluster_name = self.cluster.name
+        mgr_cluster = manager_tool.get_cluster(cluster_name)
+        if not mgr_cluster:
+            self.log.debug("Could not find cluster : {} on Manager. Adding it to Manager".format(cluster_name))
+            ip_addr_attr = 'public_ip_address' if self.cluster.params.get('cluster_backend') != 'gce' and \
```

Comment:
Why isn’t it just use what we have in node.ip_address?

Reply:
when running it from my pc, using public IP i got the private IP instead the public one...

Reply:
Should be changed, wither ip_addr or external_addr whatever needed here.

Reply:
@fgelcer this are the addresses passed to the manager, they are no used from test, node.ip_address should be good enough.